ember-engines-router-service/src/services/engine-router-service.js

```diff
@@ -65,6 +65,24 @@ export default class EngineRouterService extends Service.extend(Evented) {
     return this._externalRoutes[externalRouteName];
   }
 
+  refresh(routeName = this.currentRouteName, ...args) {
```

Comment:
this approach might be able to solve #74 as well @SergeAstapov if you want I can add it as well

Reply:
@aklkv sure, feel free to open separate PR, to keep each PR focused on one thing

Reply:
I agree with @SergeAstapov here, let's open another PR for this purpose